packages/shared/lib/models/Provider.ts

```diff
@@ -72,6 +74,10 @@ export interface IntegrationWithCreds extends Integration {
     webhook_url?: string;
     syncs: SyncConfig[];
     actions: Action[];
+    created_at: Date;
+    connections: NangoConnection[];
+    docs: string;
+    connectionCount: number;
```

Comment:
I would personnaly prefer that everything was `camelCase` but this whole object is `snake_case`, so might be better not to mix

Reply:
Updated